.github/ISSUE_TEMPLATE/✨feature.yml

```
@@ -0,0 +1,22 @@
+      required: true
+  - type: textarea
+    attributes:
+      label: ✅ 작업할 내용
+      description: 할 일을 체크박스 형태로 작성해주세요.
+      placeholder: 최대한 세분화 해서 적어주세요!
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: 🙋🏻 참고 자료
+      description: 참고 자료가 있다면 작성해 주세요.
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)